easybuild/easyconfigs/p/PMIx/PMIx-3.1.4-GCCcore-9.2.0.eb

```diff
@@ -0,0 +1,47 @@
+# Note:
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+easyblock = 'ConfigureMake'
+
+name = 'PMIx'
+version = '3.1.4'
+
+homepage = 'https://pmix.org/'
+description = """Process Management for Exascale Environments
+PMI Exascale (PMIx) represents an attempt to
+provide an extended version of the PMI standard specifically designed
+to support clusters up to and including exascale sizes. The overall
+objective of the project is not to branch the existing pseudo-standard
+definitions - in fact, PMIx fully supports both of the existing PMI-1
+and PMI-2 APIs - but rather to (a) augment and extend those APIs to
+eliminate some current restrictions that impact scalability, and (b)
+provide a reference implementation of the PMI-server that demonstrates
+the desired level of scalability.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '9.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/pmix/pmix/releases/download/v%(version)s']
```

Comment:
`s/pmix/openpmix/g`?

Reply:
Ah yes, although both work, since pmix/pmix redirects to openpmix/openpmix.
That's why I didn't notice.